keras_cv/models/generative/stable_diffusion/stable_diffusion.py

```diff
@@ -0,0 +1,177 @@
+import numpy as np
+from tqdm import tqdm
+import math
+
+import tensorflow as tf
+from tensorflow import keras
+
+from decoder import Decoder
+from diffusion_model import DiffusionModel
```

Comment:
from keras_cv.models.generative.stable_diffusion.decoder import Decoder
from keras_cv.models.generative.stable_diffusion.diffusion_model import DiffusionModel
from keras_cv.models.generative.stable_diffusion.text_encoder import TextEncoder
from keras_cv.models.generative.stable_diffusion.clip_tokenizer import SimpleTokenizer
from keras_cv.models.generative.stable_diffusion.constants import _UNCONDITIONAL_TOKENS, _ALPHAS_CUMPROD

Reply:
```python
from keras_cv.models.generative.stable_diffusion.text_encoder import TextEncoder
from keras_cv.models.generative.stable_diffusion.clip_tokenizer import SimpleTokenizer
```

Are this not NLP classes?